sdk/openai/azure-ai-openai/src/samples/java/com/azure/ai/openai/GetCompletionsAsync.java

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.openai;
+
+import com.azure.ai.openai.models.Choice;
+import com.azure.ai.openai.models.CompletionsOptions;
+import com.azure.ai.openai.models.CompletionsUsage;
+import com.azure.core.credential.AzureKeyCredential;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Sample demonstrates how to get completions for the provided input prompts. Completions support a wide variety of
+ * tasks and generate text that continues from or "completes" provided prompt data.
+ */
+public class GetCompletionsAsync {
+    /**
+     * Runs the sample algorithm and demonstrates how to get completions for the provided input prompts.
+     * Completions support a wide variety of tasks and generate text that continues from or "completes" provided
+     * prompt data.
+     *
+     * @param args Unused. Arguments to the program.
+     */
+    public static void main(String[] args) throws InterruptedException {
+        String azureOpenaiKey = "{azure-open-ai-key}";
+        String endpoint = "{azure-open-ai-endpoint}";
+        String deploymentOrModelId = "{azure_open_ai_deployment_model_id}";
```

Comment:
we should use `-` to be consistent with the endpoint and key pattern.

```suggestion
        String deploymentOrModelId = "{azure-open-ai-deployment-model-id}";
```